src/mapgen/mapgen.cpp

```diff
@@ -717,8 +722,8 @@ void MapgenBasic::generateBiomes()
 
 				depth_top = biome->depth_top;
 				base_filler = MYMAX(depth_top +
-					biome->depth_filler +
-					noise_filler_depth->result[index], 0.0f);
+					biome->depth_filler + (noise_filler_depth ?
+					noise_filler_depth->result[index] : 0.0f), 0.0f);
```

Comment:
the `noise_filler_depth ? noise_filler_depth->result[index] : 0.0f` does not need to be done more than once and should be moved into a variable in the parent scope

Reply:
true, will fix that too, thanks